modules/client-handler/src/main/java/org/apache/ignite/client/handler/configuration/ClientHandlerConfigurationModule.java

```diff
@@ -35,7 +34,7 @@ public ConfigurationType type() {
     }
 
     @Override
-    public Collection<RootKey<?, ?>> rootKeys() {
-        return Collections.singleton(ClientConnectorConfiguration.KEY);
+    public Collection<Class<?>> schemaExtensions() {
+        return List.of(ClientConnectorExtensionConfigurationSchema.class);
```

Comment:
Is it necessary to create an almost-empty `ClientConnectorExtensionConfigurationSchema` class? Can we somehow pass `ClientConnectorConfigurationSchema` directly?

Reply:
It just looks like there must be a shortcut to avoid manually creating an empty class, but I might be wrong.

Reply:
Ok, let's keep it if there is no easy way to simplify.